default.cfg.example

```diff
@@ -125,3 +125,24 @@ hideCoins = True
 #maxtolend = 0
 #maxpercenttolend = 0
 #maxtolendrate = 0
+
+[Notifications]
+notify_tx_coins = True
+notify_xday_threshold = True
+notify_summary_minutes = 60
+notify_caught_exception = True
+
+email = True
```

Comment:
Should probably set these to `False` by default, in case someone wants to do minimal configuring.